client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -204,24 +208,61 @@ class ProductForm extends Component {
 						label={ translate( 'Price' ) }
 						component={ renderPriceField }
 					/>
-					<ReduxFormFieldset name="multiple" type="checkbox" component={ CompactFormToggle }>
-						{ translate( 'Allow people to buy more than one item at a time.' ) }
-					</ReduxFormFieldset>
-					<ReduxFormFieldset
-						name="email"
-						label={ translate( 'Email' ) }
-						explanation={ translate(
-							'This is where PayPal will send your money.' +
-								" To claim a payment, you'll need a {{paypalLink}}PayPal account{{/paypalLink}}" +
-								' connected to a bank account.',
-							{
-								components: {
-									paypalLink: <ExternalLink href="https://paypal.com" target="_blank" />,
-								},
-							}
-						) }
-						component={ FormTextInput }
-					/>
+					{ config.isEnabled( 'memberships' ) && (
+						<ReduxFormFieldset name="recurring" type="checkbox" component={ CompactFormToggle }>
+							{ translate( 'Make this product a recurring subscription.' ) }
+						</ReduxFormFieldset>
+					) }
+					{ ! this.props.isRecurringSubscription && (
+						<div>
+							<ReduxFormFieldset name="multiple" type="checkbox" component={ CompactFormToggle }>
+								{ translate( 'Allow people to buy more than one item at a time.' ) }
+							</ReduxFormFieldset>
+							<ReduxFormFieldset
+								name="email"
+								label={ translate( 'Email' ) }
+								explanation={ translate(
+									'This is where PayPal will send your money.' +
+										" To claim a payment, you'll need a {{paypalLink}}PayPal account{{/paypalLink}}" +
+										' connected to a bank account.',
+									{
+										components: {
+											paypalLink: <ExternalLink href="https://paypal.com" target="_blank" />,
+										},
+									}
+								) }
+								component={ FormTextInput }
+							/>
+						</div>
+					) }
+					{ this.props.isRecurringSubscription && (
+						<div>
+							<QueryMembershipsConnectedAccounts />
+							<ReduxFormFieldset
+								name="stripe_account"
+								explanation={ translate(
+									'This is the Stripe Account where the funds will end up.'
+								) }
+								label={ translate( 'Stripe Account' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Stripe account' )` *ES Score: 18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/new/memberships-v1)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).